biostation/src/exceptions/common/mod.rs

```diff
@@ -5,13 +5,12 @@ mod syscall;
 global_asm!(include_str!("handler.S"));
 
 extern "C" {
-    #[used]
```

Comment:
This throws `error: attribute must be applied to a `static` variable` and the variable is used below, so I think it's okay to remove it. Same for the other occurrence of `#[used]`.

Reply:
Sure, seems good.